src/open_order.py

```diff
@@ -1,11 +1,55 @@
 from __future__ import annotations
```

Comment:
Seems weird to have this as a standalone module. Maybe we can have an orderbook.py module?